client/api/assignment.py

```diff
@@ -428,21 +427,21 @@ def _load_parsons(self):
         """Verifies that all desired parsons problems exist and that the 
         structure of parsons is proper.
         """
-        if self.parsons is core.NoValue:
+        if self.fpp is core.NoValue:
             return
         log.info('Loading parsons problems')
-        for prob_group_name, v in self.parsons.items():
+        for prob_group_name, v in self.fpp.items():
             req_probs = v.get('required', []) 
             opt_probs = v.get('optional', []) 
             if 'required' not in v and 'optional' not in v:
-                error_message = "At least one of required key or optional key must be in parsons problem group"
+                error_message = 'Need a "required" and/or "optional" key in an fpp config object'
                 raise ex.LoadingException(error_message)
             if not isinstance(req_probs, list) or not isinstance(opt_probs, list): 
-                error_message = "Required or Optional value from parsons problem group must be a list" 
+                error_message = 'The "required" and "optional" keys, if included in an fpp config object, must be lists'
                 raise ex.LoadingException(error_message)
             for prob in (req_probs + opt_probs):
                 if prob not in self.test_map:
-                    error_message = f"Problem name '{prob}' in parsons group '{prob_group_name}' is invalid" 
+                    error_message = f'Problem name "{prob}" in the parsons group "{prob_group_name}" is invalid' 
```

Comment:
"in fpp config object" now?

Reply:
fpp config object is different from a parsons group. I can change the name "parsons group" to "fpp problem group" for clarity. An example "fpp config object" (lmk if there is a better name for this) in the .ok file is:
`"fpp": {
    "prob group 1": {
        "required": ["add_in_range"],
        "optional": ["add_ind_if_odd", "count_until_larger"]
    },
    "prob group 2": {
        "required": ["digit_pos_match"],
        "optional": ["k_in_num"]
    }
}
`

Reply:
ooh okay, i see -- yeah, "fpp problem group" seems more consistent then.